app/controllers/api/storage.php

```diff
@@ -904,23 +905,12 @@
             throw new Exception(Exception::STORAGE_FILE_NOT_FOUND);
         }
 
-        $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $project->getId() . DIRECTORY_SEPARATOR . $bucketId . DIRECTORY_SEPARATOR . $fileId)); // Limit file number or size
-        $data = $cache->load($key, 60 * 60 * 24 * 30 * 3/* 3 months */);
-
```

Comment:
If we removed this, where are we fetching the cached files from ?

Reply:
api shutdown hook

Reply:
okay 👍